grails-app/controllers/au/org/ala/biocollect/merit/ProjectController.groovy

```diff
@@ -602,6 +602,13 @@ class ProjectController {
             }
         }
 
+        values.externalIds = values.externalIds.collect { idMap ->
```

Comment:
don't think this is required since project is already passing external ids

Reply:
@temi but there's no project being passed in the payload

Reply:
@temi I've removed that snippet now and pushed the updates to the repo